roles/generate-jenkins/templates/Jenkinsfile.j2

```diff
@@ -391,7 +391,7 @@ pipeline {
 {% if github_project_name != "docker-jenkins-builder" %}
               docker pull linuxserver/jenkins-builder:latest
 {% endif %}
-              docker run --rm -e CONTAINER_NAME=${CONTAINER_NAME} -e GITHUB_BRANCH={{ ls_branch }} -v ${TEMPDIR}:/ansible/jenkins {% if github_project_name != "docker-jenkins-builder" %}linuxserver/jenkins-builder:latest{% else %}jenkinslocal:${COMMIT_SHA}-${BUILD_NUMBER}{% endif %} 
```

Comment:
Check for these spaces they are needed for templating

Reply:
Would this work?
```suggestion
docker run --rm -e CONTAINER_NAME=${CONTAINER_NAME} -e GITHUB_BRANCH={{ ls_branch }} -v ${TEMPDIR}:/ansible/jenkins {% if github_project_name != "docker-jenkins-builder" %}linuxserver/jenkins-builder:latest{% else %}jenkinslocal:${COMMIT_SHA}-${BUILD_NUMBER}{% endif %} {{ ' ' }}
```
Ref https://jinja.palletsprojects.com/en/2.11.x/templates/#escaping

Reply:
not sure you can build and template locally

Reply:
it is about the line break though not the space

Reply:
I'll test add report.

Reply:
Tested and confirmed.